clio-util/src/main/scala/org/broadinstitute/clio/util/AuthUtil.scala

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.clio.util
+
+import akka.http.scaladsl.model.headers.OAuth2BearerToken
+import de.heikoseeberger.akkahttpcirce.ErrorAccumulatingCirceSupport
+import io.circe.parser.decode
+import org.broadinstitute.clio.util.json.ModelAutoDerivation
+import org.broadinstitute.clio.util.model.ServiceAccount
+
+import scala.io.Source
+import scala.sys.process.Process
+
+object AuthUtil extends ModelAutoDerivation with ErrorAccumulatingCirceSupport {
+
+  /** Scopes needed from Google to get past Clio's auth proxy. */
+  private val authScopes = Seq(
+    "https://www.googleapis.com/auth/userinfo.profile",
+    "https://www.googleapis.com/auth/userinfo.email"
+  )
+
+  def getBearerToken(serviceAccountPath: Option[String]): OAuth2BearerToken = {
```

Comment:
Instead of returning the `OAuth2BearerToken`, could you return a `GoogleCredentials`? Then you won't need to add akka-http as a dependency to the util project.